ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/DataMapper.java

```diff
@@ -258,4 +258,16 @@ public void closeBucket(String bucketName) {
             getRootElement().hover();
         }
     }
+
+    public void addConstant(String value, String type) {
+        SelenideElement constantsDiv = getRootElement().$(By.id("Constants")).shouldBe(visible);
```

Comment:
My observation is that by convention we add all selectors to a Element inner class